org.mixedrealitytoolkit.spatialmanipulation/Tests/Runtime/BoundsControlTests.cs

```diff
@@ -299,7 +299,7 @@ public IEnumerator TestNoHandlesToggleWhenMovingWithObjectManipulator([ValueSour
             Assert.IsFalse(objectManipulator.IsGrabSelected, "ObjectManipulator should have been released!");
             Assert.IsFalse(bc.HandlesActive, "Handles should not have been toggled.");
 
-            TestUtilities.AssertAboutEqual(bc.transform.position, initialObjectPosition, $"Object should be placed generally in the same position! Actual position: {bc.transform.position:F5}, should be {initialObjectPosition}", 0.00001f);
+            TestUtilities.AssertAboutEqual(bc.transform.position, initialObjectPosition, $"Object should be placed generally in the same position! Actual position: {bc.transform.position:F5}, should be {initialObjectPosition}", 0.01f);
```

Comment:
Doesn't seem related to this change, was it not passing without this?

Reply:
Exactly, wasn't passing, here is a screenshot of the failure:
<img width="878" alt="image" src="https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/2c2640a4-034e-4169-a437-1419d86fc547">